lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_release.rb

```diff
@@ -52,7 +53,13 @@ def self.details
       end
 
       def self.available_options
-        # Define all options your action supports.
+        [
+          FastlaneCore::ConfigItem.new(key: :default_branch,
+                                       env_name: 'FL_RELEASE_TOOLKIT_DEFAULT_BRANCH',
+                                       description: 'Default branch of the repository',
+                                       is_string: false, # true: verifies the input is a string, false: every kind of value
```

Comment:
Same as above:
```suggestion
                                       type: String,
```

(I'll let you fix all occurrences of that in all other actions rather than comment on every single one of them with a GitHub suggestion 😛 )

Reply:
But if you did, I'd just be able to apply your suggestions instead 😛